app/models/jwt_codec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Encodes/decodes JSON Web Tokens
+class JwtCodec
```

Comment:
Are you sure this is the model, not service?

Reply:
There is no such thing as service in MVC :)

Reply:
Seriously, I want to build an object-oriented design in this project so I rejected idea of "service layer" (interactors, service objects, etc). They are just disguised procedures.